starter-code/src/viking.js

```diff
@@ -1,11 +1,207 @@
+var weapons = ['gun', 'hammer', 'granade', 'bazooka', 'knife', 'shotgun', 'sniper', 'assault rifle', 'sword', 'missile', 'revolver'];
 
-// Viking
-function Viking() {}
+var createViking = document.getElementById('createViking');
+var createSaxon  = document.getElementById('createSaxon');
+var attackViking = document.getElementById('attackViking');
 
-// Saxon
-function Saxon() {}
+var attackSaxon  = document.getElementById('attackSaxon');
+var result       = document.getElementById('result');
+var vikingDraw    = document.getElementById('vikingDraw');
+var saxonDraw    = document.getElementById('saxonDraw');
 
-// War
-function War() {}
+
+
+// SOLDIER
+function Soldier(health, strength) {
+    this.health = health;
+    this.strength = strength;
+}
+
+Soldier.prototype.attack = function(){
+    return this.strength;
+};
+
+Soldier.prototype.receiveDamage = function(damage){
+    this.health -= damage;
+};
+
+//////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+
+
+//VIKING
+function Viking(name, health, strength, randomWeapon, team) {
+    Soldier.call(this, health, strength);
+    this.name = name;
+    this.strength = strength;
+    this.randomWeapon = randomWeapon;
+    this.team = team;
+}
+Viking.prototype = Object.create(Soldier.prototype);
+Viking.prototype.constructor = Viking;
+
+
+Viking.prototype.receiveDamage = function(damage) {
+    this.health -= damage;
+    
+    if (this.health > 0) {
+        return this.name + ' has received ' + damage + ' points of damage';
+    } else {
+        return this.name + ' has died in act of combat';
+    }
+};
```

Comment:
Con esto machacas la función heredada, tienes que hacer una llamada a `Soldier.prototype.receiveDamage.call(this, damage)` dentro de `Viking.prototype.receiveDamage(damage)`